.validate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "scripts": {
+    "lint": "jshint ."
+  },
+  "pre-commit": ["lint", "validate", "test"]
+}
```

Comment:
This was what an `npm install` added manually so I went with it.  Updating precommit-hook is outside the scope of what I'm trying to do here, doing my best to avoid a rabbit hole of updating here.